### PR TITLE
Update whatsapp to 0.2.9230

### PR DESCRIPTION
**Pull request by nicoalonsop**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/whatsapp.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'whatsapp' do
-  version '0.2.9229'
-  sha256 '1ddef708bbea02d4f04f11956c0b6d77ea02fdf5162085855f89530a7a6352a9'
+  version '0.2.9230'
+  sha256 '116c0fe2398fd4763bfe97044f9a655e764abdd3703f5c57edcd7e672279e1de'
 
   url "https://web.whatsapp.com/desktop/mac/files/release-#{version}.zip"
```
Comment (victorpopkov) [inline Casks/whatsapp.rb]: Both appcast and download URL point to the v0.2.9229 as the latest.

**Comment by brianmorton** [comment]
I too am interested in how you found this version...

**Comment by nicoalonsop** [comment]
The app has autoupdated to that version, also in AppStore it is in that version (0.2.9230)

**Comment by brianmorton** [comment]
https://github.com/Homebrew/homebrew-cask/pull/48381

**Comment by commitay** [comment]
Closing in favour of https://github.com/Homebrew/homebrew-cask/pull/48381

**Pull request closed without merging**